dags/old_worker_single_notice_process_orchestrator.py

```diff
@@ -209,16 +209,17 @@ def _validate_transformed_rdf_manifestation(**context_args):
         mongodb_client = MongoClient(config.MONGO_DB_AUTH_URL)
         notice_repository = NoticeRepository(mongodb_client=mongodb_client)
         mapping_suite_repository = MappingSuiteRepositoryMongoDB(mongodb_client=mongodb_client)
+        validate_xpath_coverage_notice_by_id(notice_id=notice_id, mapping_suite_identifier=mapping_suite_id,
```

Comment:
Why is this in old_worker_single_notice_process.....py file? do we have a new one?

Reply:
Should this be in the other file ?

Reply:
it's the refactoring Stefan is doing.
I think it will be aligned when he finishes he's game.